packages/master-detail-layout/src/vaadin-master-detail-layout.js

```diff
@@ -394,6 +436,16 @@ class MasterDetailLayout extends ResizeMixin(ElementMixin(ThemableMixin(PolylitM
       this.setAttribute('overlay', '');
     }
   }
+
+  /** @private */
+  __getStackThreshold() {
```

Comment:
Maybe a bit unfortunate that we have to do this every time when the mode is detected. But I guess caching is not an option when using relative units (em, rem, etc.)?

Reply:
You can slightly optimize this by setting `background-position` only once when the threshold changes. This should avoid forcing a style recalculation every time, see the approach we used in form-layout:

https://github.com/vaadin/web-components/blob/056c1efbf5bb23ed5aa6606a23a80157aef2f10e/packages/form-layout/src/vaadin-form-layout-styles.js#L103-L105

https://github.com/vaadin/web-components/blob/056c1efbf5bb23ed5aa6606a23a80157aef2f10e/packages/form-layout/src/layouts/auto-responsive-layout.js#L168-L171

Reply:
Thanks, updated to use this approach with pseudo-element and a custom property.